lib/entity/config/config_list_item.dart

```diff
@@ -5,15 +5,23 @@ import 'package:seeder/controls/doc_field_range_slider.dart';
 import 'package:seeder/controls/doc_field_slider.dart';
 import 'package:seeder/controls/group.dart';
 import 'package:seeder/providers/firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
 
 class ConfigListItem extends ConsumerWidget {
+  final String currentAuthor = FirebaseAuth.instance.currentUser!.uid;
   final String path;
+  String author="";
   final String entityId;
```

Comment:
Instead of entityId let's pass through entityDocument which contains author that will not change.

Reply:
I have checked the "selectedEntityDoc" in the config_list.dart and it is mostly null, or does not have the field "author". I don't know is it the entityDocument you are looking for?